### PR TITLE
`add liquidity` - faq screen

### LINKED ISSUE #561: `add liquidity` - faq screen
#### What would you like to be added:
Need to add FAQ screen for add liquidity

```
What is liquidity mining?

Liquidity mining is a DeFi mechanism in which participants supply cryptocurrencies into liquidity pools, and in turn be rewarded with fees and tokens based on their share of the total pool liquidity. Liquidity pools in DeFiChain consist of liquidity in pairs of coins, used by the DeFiChain DEX (Decentralized Exchange).

How do I supply liquidity?

After deciding which pool pair you would like to supply liquidity to, click ADD LIQUIDITY and enter the input amount for either of the tokens in the pair. Once an amount is entered on one side of the pair, the app automatically balances the amount of the other side, based on the ratio of current liquidity in the pool.

How do I send tokens from my DFI wallet?

Tokens on DeFiChain can only be sent to DFI addresses only. This means that you should not send directly to an exchange or a third party platform that does not support DeFiChain. Similarly, liquidity tokens can only be sent to another DFI wallet address.

What are some exchanges and platforms I can send to?

1. Kucoin
2. Bittrex
3. Transak
4. Cake DeFi

View more at DeFiChain.com

Can I send liquidity pool tokens to an exchange or another entity?

Liquidity tokens can only be sent to another DFI wallet address, and not to an exchange of central entity.

How do I receive tokens on my DFI wallet?

You can receive DFI tokens into your wallet by sharing your address found in the Receive page, or by getting the sender to scan the QR code. Note that you will not be able to receive native tokens (e.g. BTC, ETH) directly into your wallet.
#### Why is this needed:

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What kind of PR is this?:

/kind

#### What this PR does / why we need it:

- add liquidity mining faq to knowledge base page

#### Which issue(s) does this PR fixes?:
 
Fixes #561 

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Comment by kyleleow** [comment]
To automatically link an github issue, just type the github issue number in the PR description, e.g.: `Fixes #561`

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      title: translate('components/LiquidityMiningFaq', 'How do I send tokens from my DFI wallet?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', 'Tokens on DeFiChain can only be sent to DFI addresses only. This means that you should not send directly to an exchange or a third party platform that does not support DeFiChain. \n\nSimilarly, liquidity tokens can only be sent to another DFI wallet address.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/LiquidityMiningFaq', 'What are some exchanges and platforms I can send to?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', '1. Kucoin\n2. Bittrex\n3. Transak\n4. Cake DeFi \n\nView more on [DeFiChain.com](https://defichain.com/dfi/)'),
```
Comment (pierregee) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx:25]: Is it suppose to be a link just like this?

View more on [DeFiChain.com](https://defichain.com/dfi/)

File: mobile-app/cypress/integration/functional/wallet/settings/knowledgeBase.spec.ts [hunk 2]
```diff
@@ -56,6 +56,19 @@ context('Wallet - Settings', () => {
     cy.url().should('include', 'app/Settings/KnowledgeBaseScreen')
   })
 
+  it('should navigate to DEX faq from knowledge base page', function () {
+    cy.getByTestID('dex_faq').should('exist').click()
+    cy.url().should('include', 'app/Settings/DexFaq')
+    cy.go('back')
+    cy.url().should('include', 'app/Settings/KnowledgeBaseScreen')
+  })
+  it('should navigate to Liquidity Mining faq from knowledge base page', function () {
```
Comment (fullstackninja864) [inline mobile-app/cypress/integration/functional/wallet/settings/knowledgeBase.spec.ts:65]:
```suggestion

  it('should navigate to Liquidity Mining faq from knowledge base page', function () {
```

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      }]
+    },
+    {
+      title: translate('components/LiquidityMiningFaq', 'How do I receive tokens on my DFI wallet?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', 'You can receive DFI tokens into your wallet by sharing your address found in the Receive page, or by getting the sender to scan the QR code. \n\nNote that you will not be able to receive native tokens (e.g. BTC, ETH) directly into your wallet.'),
+        type: 'paragraph'
+      }]
+    }
+
```
Comment (fullstackninja864) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx:43]:
```suggestion
```

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      }]
+    },
+    {
+      title: translate('components/DexFaq', 'What is slippage tolerance?'),
+      content: [{
+        text: translate('components/DexFaq', 'Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in **slippage**, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.'),
+        type: 'paragraph'
+      }]
+    }
+
```
Comment (fullstackninja864) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx:22]:
```suggestion
```

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/KnowledgeBaseScreen.tsx [hunk 5]
```diff
@@ -28,7 +28,18 @@ export function KnowledgeBaseScreen ({ navigation }: Props): JSX.Element {
...
+      label: 'DEX',
+      testID: 'dex_faq',
+      onPress: () => navigation.navigate('DexFaq')
+    },
+    {
+      label: 'Liquidity Mining',
+      testID: 'liquidity_mining_faq',
+      onPress: () => navigation.navigate('LiquidityMiningFaq')
     }
+
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Settings/screens/KnowledgeBaseScreen.tsx:42]:
```suggestion
```

File: mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx [hunk 6]
```diff
@@ -226,6 +228,33 @@ export function SettingsNavigator (): JSX.Element {
...
+          headerTitle: () => (
+            <HeaderTitle
+              text={translate('screens/WalletNavigator', 'DEX FAQ')}
+              containerTestID={headerContainerTestId}
+            />
+          ),
+          headerBackTitleVisible: false
+        }}
+      />
+      <SettingsStack.Screen
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx:244]:
```suggestion

      <SettingsStack.Screen
```

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+      title: translate('components/DexFaq', 'Transaction fees'),
+      content: [{
+        text: translate('components/DexFaq', 'Each transaction will be subject to a small amount of fees. The amount may vary depending on the network\'s congestion.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/DexFaq', 'What is slippage tolerance?'),
+      content: [{
+        text: translate('components/DexFaq', 'Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in **slippage**, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.'),
```
Comment (fullstackninja864) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx:18]: `slippage` should be in bold

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx (hunk 1 above)
Comment (chloezxyy) [reply mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx]: Resolved with the wallet team

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx (hunk 7 above)
Comment (chloezxyy) [reply mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx]: resolved with the wallet team

**Review by kyleleow — APPROVED** [review]

**Pull request closed without merging**